### PR TITLE
Implemented "readOnly" property (mobile friendly feature)

### PR DESCRIPTION
**Pull request by AndreaCatania**
Added the property "readOnly", this is used to block user input and prevent keyboard opening on mobile phones.
Ofcourse default value is false and the input is enabled on readOnly true.

This feature was requested some time ago, by other guys #110 today i've implemented this feature because I want to use this (fantastic) library for my project and this feature is essential.

### CODE REVIEW

**Comment by AndreaCatania** [comment]
What's wrong?

**Comment by AndreaCatania** [comment]
Why coverage fails? What's wrong in the code? I've tried in local and works fine.

**Comment by AndreaCatania** [comment]
With this coverage i'm forced to write a bad code instead of one "IF" I had to write two "IF"... absolutely useless

**Comment by drewcovi** [comment]
:+1:

**Comment by samstickland** [comment]
+1

**Comment by kayluhb** [comment]
:+1:

**Comment by filiplikavcan** [comment]
+1 This really fixes undesired behaviour on touch devices where keyboard widget shows up when user touches 'selectized select. Thank you.

**Comment by joseluisq** [comment]
:+1:

File: src/selectize.js [hunk 1]
```diff
@@ -126,6 +126,9 @@ $.extend(Selectize.prototype, {
 		$dropdown         = $('<div>').addClass(settings.dropdownClass).addClass(inputMode).hide().appendTo($dropdown_parent);
 		$dropdown_content = $('<div>').addClass(settings.dropdownContentClass).appendTo($dropdown);
 
+		$control.addClass(self.settings.readOnly?'input-readonly':'');
+		$control_input.attr('readonly', self.settings.readOnly);
```
Comment (amakhrov) [inline src/selectize.js:130]: shouldn't it be `.prop('readonly')` instead of `.attr('readonly')`?

**Comment by amakhrov** [comment]
This PR probably can't be accepted as is.
README:

> When issuing a pull request, please exclude changes in the "dist" folder to avoid merge conflicts.

**Comment by joallard** [comment]
1. Would you please remove changes in dist, as stated?
2. Can you tell me why we're not using attr 'required' instead of adding an option?
3. Can you add tests?

**Pull request closed without merging**